### PR TITLE
OSDOCS-2532: Add overview page for a support 

### PR DESCRIPTION
**Pull request by xenolinux**
- [Jira issue](https://issues.redhat.com/browse/OSDOCS-2532)
- [Preview](https://deploy-preview-39256--osdocs.netlify.app/openshift-enterprise/latest/support/)
- OCP version: 4.6+
- QA contacts: 
Storage: @duanwei33 kindly review the bullet point `Storage issues`- LGTM'd
Monitoring: @lihongyan1 @juzhao  kindly review the bullet point `Monitoring issues` and `Remote heath monitoring issues` -LGTM'd (monitoring issues) LGTM'd
Network: @zhaozhanqi kindly review the bullet point `Network issues` and `A network trace` - LGTM'd
Logging: @IshwarKanse kindly review the bullet points `Bootstrap node journal logs`, `Cluster node journal logs` - LGTM'd
Installation: @pamoedom kindly review the bullet point `installation issues`; also `Gather data about your cluster`- LGTM'd
Crio and Node: @sunilcio kindly review the bullet points  `Node issues` and `Crio issues` - LGTM'd

### CODE REVIEW

**Comment by pamoedom** [comment]
Hi @xenolinux, apologies for the delay, here you have my comments:
- Regarding `Installation issues`, that section seems scarce to me, it points to the big troubleshoot docs so I think we should modify it adding at least the following points:
~~~
- Monitor the different installation stages.
- Determine where installation issues occur.
- Investigate multiple installation issues.
- Gather logs from a failed installation.
~~~
- Regarding `Gather data about your cluster`, it looks ok to me with one exception, the latest entry `Toolbox container`, I think it would be better to remove it, because is not a gather method by itself, is a tool used precisely to run other commands within a container, like `sosreport` or `redhat-support-tool`, what do you think?

Best Regards.

**Comment by xenolinux** [comment]
> Thanks for the review. I changed the points in `Installation issues` section as suggested.
About the toolbox,  I agree with removing it from the list. Changed that too.

**Comment by IshwarKanse** [comment]
@xenolinux LGTM

**Comment by xenolinux** [comment]
@pamoedom Does those sections look good? If yes would you please add the LGTM comment? Or Let me know if anything else needs a change.

**Comment by zhaozhanqi** [comment]
@xenolinux LGTM

File: _topic_maps/_topic_map.yml [hunk 1]
```diff
@@ -456,6 +456,8 @@ Name: Support
 Dir: support
 Distros: openshift-enterprise,openshift-online,openshift-origin
 Topics:
+- Name: Support overview
+  File: index
```
Comment (kmccarron-rh) [inline _topic_maps/_topic_map.yml:460]: test comment

**Comment by juzhao** [comment]
`Monitoring issues` LGTM,  for `Remote heath monitoring issues` part, there is Insight Operator needs the insight team help to check, @JoaoFula @quarckster can you help to check the Insight part?

**Comment by lihongyan1** [comment]
For monitoring, suppose the following should be links
*** Investigate why user-defined metrics are unavailable
*** Determine why Prometheus is consuming a lot of disk space

**Comment by juzhao** [comment]
> For monitoring, suppose the following should be links *** Investigate why user-defined metrics are unavailable *** Determine why Prometheus is consuming a lot of disk space

good catch, and checked,  for each logging issue, link is added, but for other components, link is only added for "xxx issues"

**Comment by juzhao** [comment]
no detail oc issues under `OpenShift CLI (oc) issues:` part, cc: @zhouying7780 please help to check CLI part

**Comment by mjpytlak** [comment]
@xenolinux Please add the BZ # to the title of the PR. Also, I have completed my peer review, but because this my first time on the squad, someone else is going to come in behind me to review, as well. You can find my comments [here](https://docs.google.com/document/d/1pcR64hXRcgpxDDFRV21FImvzTehQj1hEYsE8Tl5vNXk/edit), but I suggest holding off on updates until the other member has had a chance to weigh in.

**Comment by xenolinux** [comment]
Build is failing with the msg: `xml:id : attribute value ”support-overview-gather-data-cluster_support-overview” is not an NCName, line 66, column 79 (master.xml, line 66)`

**Comment by xenolinux** [comment]
Closing-reopening PR to start the build again

**Comment by mjpytlak** [comment]
@xenolinux A few items:
@sagidlow please keep me honest.

1. An assembly can only have a single level 1 heading. If I am reading this correctly, all headings should be adjusted to a level.
2. An anchor ID in an assembly does not require `_{context}`. The latter is only required in modules. Please remove all instances of `_{context}` from the heading anchors.
3. With the addition of the headings, please adjust your bullet levels. I did not notice this the first time through but you are using `+` to separate bullet levels. I do not believe this is necessary, as the style sheet associated with the build adds the correct amount of whitespace.

Current
`** *Telemetry*: The Telemetry Client gathers and uploads the metrics values to Red Hat every four minutes and thirty seconds. Red Hat uses this data to:

*** Monitor the clusters.
*** Roll out {product-title} upgrades.
*** Improve the upgrade experience.`

Fixed
`* *Telemetry*: The Telemetry Client gathers and uploads the metrics values to Red Hat every four minutes and thirty seconds. Red Hat uses this data to:

** Monitor the clusters.
** Roll out {product-title} upgrades.
** Improve the upgrade experience.`

**Comment by sagidlow** [comment]
@mjpytlak That's correct. There can only be one level one heading in an assembly. You would have to do these sections has level 2 headings. I think my thought process when I added the comments for the headings was to have them be their own standing modules. But that might make linking a bit difficult. You are correct about dropping the {_context} from the ID statements. I also agree with adjusting the bullets :)

**Comment by xenolinux** [comment]
> > For monitoring, suppose the following should be links *** Investigate why user-defined metrics are unavailable *** Determine why Prometheus is consuming a lot of disk space
> 
> good catch, and checked, for each logging issue, link is added, but for other components, link is only added for "xxx issues"

Every sub-bullet point cannot be added link since every sub-bullet would not have anything to point to.

**Comment by xenolinux** [comment]
@mjpytlak @sagidlow I just had a thought while looking at a preview. Would it be a good idea to include a table for all troubleshooting issues?

**Pull request merged**